### PR TITLE
DOCSP-32063 compass 1.39.1 release notes

### PR DESCRIPTION
**Pull request by kennethdyer**
## DESCRIPTION

Release notes for Compass 1.39.1.

## STAGING

[RN](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-32063-compass-1.39.1-release-notes/release-notes/#compass-1.39.1)

## JIRA

* [DOCSP-32063](https://jira.mongodb.org/browse/DOCSP-32063)

## BUILD LOG

* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64d4393267dbe910106549a7)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,22 @@ Release Notes
...
    :class: twocols
 
+|compass| 1.39.1
+----------------
+
+*Released August 9, 2023*
+
+New Features:
+
+* Show insight for unbound array (:issue:`COMPASS-6836`)
```
Comment (ianf-mongodb) [inline source/release-notes.txt:20]: I wonder if we could ask the Compass team for more detail about this first item. I am not sure from the small description or the ticket what exactly this means.

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,22 @@ Release Notes
...
+* Use modal to highlight legacy connections (:issue:`COMPASS-7072`)
+
+Bug Fixes:
+
+* Split connection storage between processes (:issue:`COMPASS-7078`)
```
Comment (ianf-mongodb) [inline source/release-notes.txt:25]: Slightly unclear what this means for this Bug Fix, could you clarify with the Compass team on both bullet points to add a bit more detail?

**Review by ianf-mongodb — COMMENTED** [review]
Hi @kennethdyer, I left a couple items for consideration. 

Also we need to [bump version in snooty.toml](https://github.com/mongodb/docs-compass/blob/master/snooty.toml#L30) -> 1.39.1

File: source/release-notes.txt [hunk 3]
```diff
@@ -10,6 +10,22 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.1
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/release-notes.txt:13]: Fixed

File: source/release-notes.txt (hunk 1 above)
Comment (kennethdyer) [reply source/release-notes.txt]: Fixed

**Review by ianf-mongodb — APPROVED** [review]

File: source/release-notes.txt (hunk 2 above)
Comment (kennethdyer) [reply source/release-notes.txt]: Fixed

**Review by ianf-mongodb — APPROVED** [review]
LGTM % we might want to link to a description for each vulnerability: https://nvd.nist.gov/vuln/detail/CVE-2023-3730

**Review by mabaasit — APPROVED** [review]

**Pull request merged**